cmd/plugins/balloons/Dockerfile

```diff
@@ -16,10 +22,21 @@ RUN go mod download
 COPY . .
 
 RUN make clean
-RUN make IMAGE_VERSION=${IMAGE_VERSION} BUILD_VERSION=${BUILD_VERSION} BUILD_BUILDID=${BUILD_BUILDID} PLUGINS=nri-resource-policy-balloons build-plugins-static
+RUN make IMAGE_VERSION=${IMAGE_VERSION} BUILD_VERSION=${BUILD_VERSION} BUILD_BUILDID=${BUILD_BUILDID} PLUGINS=nri-resource-policy-balloons DEBUG=$DEBUG V=$DEBUG build-plugins-static
+RUN if [ "$DEBUG" = 0 ]; then \
+        for d in /go/builder/build/bin/dlv /go/builder/pkg /go/builder/cmd /go/pkg /usr/local/go/src ; do \
+            mv $d $d-dont-copy 2>/dev/null; \
+            touch $d; \
+        done; \
+    fi
 
 FROM gcr.io/distroless/static
 
 COPY --from=builder /go/builder/build/bin/nri-resource-policy-balloons /bin/nri-resource-policy-balloons
+COPY --from=builder /go/builder/build/bin/dlv /bin/dlv
```

Comment:
won't it fail if the debugging wasn't enabled, i.e. when `$DEBUG != 1`

Reply:
This does not fail because of the `touch $d` above.

That is, by default (DEBUG=0) there will be zero-sized files in the image.

I wish there was a "conditional COPY" in Dockerfile, but I suppose there is none.